components/dash-core-components/package.json

```diff
@@ -92,6 +92,7 @@
     "webpack": "^5.90.0",
     "webpack-cli": "^5.1.4"
   },
+  "optionalDependencies": { "fsevents": "*" },
```

Comment:
I see the commit title "chokidar fix" but don't know what that means or what this does, can you explain?

Reply:
The error message mention that weird name:

> npm ERR! fsevents not accessible from chokidar